hack/release/packaging/docker-build-images/install-centos-build-deps

```diff
@@ -2,18 +2,23 @@
 set -x
 set -e
 
+sed -i s/^#.*baseurl=http/baseurl=http/g /etc/yum.repos.d/*.repo
+sed -i s/^mirrorlist=http/#mirrorlist=http/g /etc/yum.repos.d/*.repo
+sed -i s/mirror.centos.org/vault.centos.org/g /etc/yum.repos.d/*.repo
```

Comment:
`mirrorlist.centos.org` is gone so adjust mirrorlist to baseurl and comment out mirrorlist.